packages/ui/src/composites/wui-card-select-loader/index.ts

```diff
@@ -1,20 +1,42 @@
 import { html, LitElement } from 'lit'
-import { customElement } from 'lit/decorators.js'
+import { customElement, property } from 'lit/decorators.js'
 import '../../components/wui-shimmer'
 import { elementStyles, resetStyles } from '../../utils/ThemeUtil'
 import styles from './styles'
+import type { CardSelectType } from '../../utils/TypesUtil'
 
 @customElement('wui-card-select-loader')
 export class WuiCardSelectLoader extends LitElement {
   public static styles = [resetStyles, elementStyles, styles]
 
+  // -- State & Properties -------------------------------- //
+  @property() public type: CardSelectType = 'wallet'
+
   // -- Render -------------------------------------------- //
   public render() {
     return html`
-      <wui-shimmer width="56px" height="56px" borderRadius="xs"></wui-shimmer>
+      ${this.shimmerTemplate()}
       <wui-shimmer width="56px" height="20px" borderRadius="xs"></wui-shimmer>
     `
   }
+
+  private shimmerTemplate() {
+    if (this.type === 'network') {
+      return html` <wui-shimmer
+          type=${this.type}
+          width="48px"
+          height="54px"
+          borderRadius="xs"
+        ></wui-shimmer>
+        <svg width="48" height="53" fill="none">
```

Comment:
This is the same svg as used in wui-network-image component right, maybe let's move it to svg's folder like with icons and reuse?